src/web/src/main/java/tupperdate/web/model/chats/ChatRepository.kt

```diff
@@ -25,4 +26,12 @@ interface ChatRepository {
         content: String,
     ): Result<Unit>
 
+    suspend fun saveNewChat(
+        chat: ModelNewChat,
+    ): Result<Unit>
+
+    suspend fun updateLikes(
+        chatId: String,
+        likes: Map<String, FieldValue>,
```

Comment:
We should not expose a `FieldValue` in a repository, since it's a Firebase implementation detail.